src/components/TextFieldChip/TextFieldChips.tsx

```diff
@@ -190,6 +165,51 @@ const TextFieldChips = React.forwardRef(
       })
     }
 
+    const handleClickAway = () => {
+      if (!isFocusingRef.current) {
+        return
+      }
+      if (chipIndexEditable !== null) {
+        clearChipIndexEditable()
+        clearInputValue()
+      } else if (!clearInputOnBlur) {
```

Comment:
Not sure if everyone want this feature if `clearInputOnBlur` is false... :/

Reply:
Fair enough.